src/plugins/chromecastPlayer/plugin.js

```diff
@@ -106,18 +101,21 @@ class CastPlayer {
             return;
         }
 
-        let applicationID = userSettings.chromecastVersion();
-        if (applicationID === 'stable') applicationID = applicationStable;
-        if (applicationID === 'unstable') applicationID = applicationUnstable;
+        const apiClient = ServerConnections.currentApiClient();
+        const userId = apiClient.getCurrentUserId();
+
+        apiClient.getUser(userId).then(user => {
+            const applicationID = user.Configuration.CastReceiverId;
```

Comment:
What happens if `CastReceiverId` is null? Does the session request just error out?

Reply:
I kinda forgot about that so I'll probably need to make some changes. It should probably also disable the cast button in the play to menu.

Reply:
I've added a check to stop the cast init in case of a falsy value, this will automatically hide the cast button from the play to menu 👍